tests/mocha/server/RoundProgressTest.js

```diff
@@ -7,20 +7,14 @@ MochaWeb.testOnly(function() {
       chai.expect(makeProgress({done: 5, total:  0}).percentage()).to.equal(0);
     });
 
-    it('isComplete()', function() {
-      chai.expect(makeProgress({done:  9, total: 10}).isComplete()).to.be.false;
-      chai.expect(makeProgress({done: 10, total: 10}).isComplete()).to.be.true;
-      chai.expect(makeProgress({done: 11, total: 10}).isComplete()).to.be.false;
-    });
-    it('isOverComplete()', function() {
-      chai.expect(makeProgress({done:  9, total: 10}).isOverComplete()).to.be.false;
-      chai.expect(makeProgress({done: 10, total: 10}).isOverComplete()).to.be.false;
-      chai.expect(makeProgress({done: 11, total: 10}).isOverComplete()).to.be.true;
+    it('completeness()', function() {
+      chai.expect(makeProgress({done:  9, total: 10}).completeness()).to.equal('incomplete');
```

Comment:
I'll let you have your pretty printing here. I just wanted you to know that I am watching =)